### ISSUE TITLE
Question about size of install

### ISSUE THREAD

**Issue by WinstonN** [garris/BackstopJS#919]
Hi Hi, 

I installed backstopjs, inside a docker container, and afterwards - I wanted to have a look at the image size

```
docker history --no-trunc IMAGE
```
This caught my eye
```
0d047f43a3ff        18 minutes ago      /bin/sh -c npm install -g backstopjs            427MB 
```
Is there a way I can reduce that size drastically?

**Comment by garris** [comment]
Yes. You could propose and build some way to conditionally include/exclude phantom and chromy module dependencies.   That would be a huge help.
